app/src/main/java/org/simple/clinic/phone/Caller.kt

```diff
@@ -0,0 +1,24 @@
+package org.simple.clinic.phone
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+
+interface Caller {
+
+  fun call(context: Context, phoneNumber: String)
+
+  object UsingDialer : Caller {
+    override fun call(context: Context, phoneNumber: String) {
+      val intent = Intent(Intent.ACTION_CALL, Uri.fromParts("tel", phoneNumber, null))
```

Comment:
Isn't this supposed to be `ACTION_DIAL`?

Reply:
mCopyPasta